Documentation/configuration/metrics.rst

```diff
@@ -67,6 +67,14 @@ Datapath
 * ``datapath_conntrack_gc_duration_seconds``: Duration in seconds of the garbage
   collector process labeled by datapath and completion status.
 
+BPF
+---
+
+* ``bpf_syscall_duration_seconds``: Duration of BPF system call performed
+  * Labels: ``operation={lookup, delete, update, objPin, getNextKey, ...}``, ``outcome={success|failure}``
+* ``cilium_bpf_map_ops_total``: Number of BPF map operations performed
```

Comment:
Any particular reason the second metric above starts with `cilium_`, but the first doesn't? The metrics are coming from cilium anyway, do they need that?

Reply:
Typo